app/controllers/idv/gpo_verify_controller.rb

```diff
@@ -43,7 +43,7 @@ def create
           if result.extra[:pending_in_person_enrollment]
             redirect_to idv_in_person_ready_to_verify_url
           else
-            event, disavowal_token = create_user_event_with_disavowal(:account_verified)
+            event, _disavowal_token = create_user_event_with_disavowal(:account_verified)
```

Comment:
Should we skip the disavowal logic altogether? Same note applies below for `ReviewController` and maybe `review_profile.rake`.

```suggestion
            event = create_user_event(:account_verified)
```

Reply:
Yes, you're probably right, no point generating a disavowal token that gets completely discarded, will update the others

Reply:
So it turns out that both forms of `create_user_event` still return the tuple format

https://github.com/18F/identity-idp/blob/fe4ff4edd5fb45bf998d24e74652ee5b0b1d0f90/app/services/user_event_creator.rb#L11-L12

so we'll still need the `_disavowal_token` as a placeholder